charts/tfy-llm-gateway-infra/values.yaml

```diff
@@ -195,7 +195,7 @@ nats:
     topologySpreadConstraints:
       kubernetes.io/hostname:
```

Comment:
`kubernetes.io/hostname` a valid key for `topologySpreadConstraints` - https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/